### ISSUE TITLE
Fails with Hugo v0.20.3 and v0.20.4

### ISSUE THREAD

**Issue by AbeEstrada** [netlify/binrc#1]
I used the [master](https://github.com/netlify/binrc/tree/master) version with `go get` and it fails to download Hugo v0.20.3 and v0.20.4 the most recent version that works is v0.20.2

### Works

```bash
$ HUGO_VERSION=0.20.2 binrc install spf13/hugo
/Users/Abe/.binrc/binaries/spf13/hugo/v0.20.2/hugo
```

### Fails

```bash
$ HUGO_VERSION=0.20.3 binrc install spf13/hugo
error downloading project spf13/hugo, from https://github.com/spf13/hugo/releases/download/v0.20.3/hugo_0.20.3_Linux-64bit.tar.gz - binary doesn't seem to exist: 404
```

```bash
$ HUGO_VERSION=0.20.4 binrc install spf13/hugo
error renaming /Users/Abe/.binrc/binaries/spf13/hugo/v0.20.4/binrc-879126892/hugo_0.20.4_linux_amd64/hugo_0.20.4_linux_amd64 to /Users/Abe/.binrc/binaries/spf13/hugo/v0.20.4/hugo: rename /Users/Abe/.binrc/binaries/spf13/hugo/v0.20.4/binrc-879126892/hugo_0.20.4_linux_amd64/hugo_0.20.4_linux_amd64 /Users/Abe/.binrc/binaries/spf13/hugo/v0.20.4/hugo: no such file or directory
```

**Comment by AbeEstrada** [comment]
I noticed inconsistencies between these releases.

- 0.20.2 the binary name is `hugo_0.20.2_linux_amd64`
- 0.20.3 the tag includes a **v** before the version number `hugo_v0.20.3_Linux-64bit`
- 0.20.4 the binary name is `hugo`

**Comment by AbeEstrada** [comment]
Probably it's a better idea to find the tag name using the API.

https://api.github.com/repos/spf13/hugo/tags

**Comment by AbeEstrada** [comment]
#2 fixes where there's an extra **v** in the tag name, now the error with the v0.20.3 release is the binary name `hugo` instead of `hugo_0.20.3_linux_amd64`

**Comment by AbeEstrada** [comment]
Added to #2 the fix to use the project name if the binary name from the template doesn't exist. That change fixes these two edge cases for `hugo` versions `0.20.3` and `0.20.4`

**Comment by calavera** [comment]
fixed on #3.
